### PR TITLE
Update grant-access-to-the-security-and-compliance-center.md

### PR DESCRIPTION
**Pull request by msbemba**
Changed the steps as protection.office.com is now called the security admin center. Also added a note that they can navigate to this section from the compliance admin center.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9  Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2648

File: microsoft-365/security/office-365-security/grant-access-to-the-security-and-compliance-center.md [hunk 1]
```diff
@@ -53,6 +53,9 @@ For more information about the different permissions you can give to users in th
 
 7. Click **Save** to save the changes to the role group.
 
+> [!NOTE]
+> Admins can also navigate to the Complaince Admin Center(https://compliance.microsoft.com/homepage) and get to [Managing Permissins](https://protection.office.com/permissions) Section.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/grant-access-to-the-security-and-compliance-center.md]:
```suggestion
> Admins can also navigate to the Compliance Admin Center(https://compliance.microsoft.com/homepage) and go to the [Managing Permissions section](https://protection.office.com/permissions).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@chrisda Please merge this PR. Thanks.

**Comment by chrisda** [comment]
@msbemba, although it looks like approved this PR, I made some different changes, so it doesn't really satisfy #2648.

The topic in question is about permissions in the Security & Compliance Center, and needs to remain specific and dedicated to the Security & Compliance Center. More to the point of the 2648, we likely need a new and separate page for granting permissions to the Security Center and also the Compliance Center, and we can cross-link between them all. That way, when the Security & Compliance Center ultimately goes away, we can just eliminate the topic without any impact on the Security Center or Compliance Center topics.

**Comment by msbemba** [comment]
@chrisda That sounds good. Let me know If you need any help from me on the new document.

Thanks
Sri

**Pull request merged**